src/features/people/PersonPage/PersonDetails/index.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { theme } from '../../../../app/theme';
+import { useWindowSize } from '../../../../common/useWindowSize';
+import { Wrapper, ProductionDetails, Property, DetailsWrapper, Detail, Biography, Image, Name } from './styled';
+
+
+
+export const PersonDetails = (props) => {
+
+    const [width] = useWindowSize();
+
+    const formatBirthday = (birthdate) => {
```

Comment:
Maybe it would be better if you named this argument birthday instead of birthdate? Or even make this function a hook named useFormatDate which could be stored in common in case it was needed somewhere else.

Reply:
👍